src/lib/zork/zork.ts

```diff
@@ -359,36 +352,35 @@ const zorkServer = net.createServer((client) => {
   });
 
   client.on('end', () => {
-    console.info(`client ${ctx.clientId} disconnected`);
-    if (ctx.mode === 'give') {
-      npeersGetting--;
-      console.info('decremented npeersGetting to', npeersGetting);
+    ctx.log(`[zorkServer] client disconnected`);
+    if (ctx.mode === 'give' && ctx.rtc.conn) {
+      numGetters--;
+      ctx.log(`decremented numGetters to ${numGetters}`);
     }
```

Comment:
I looked for some type of `onclose` event I could listen for on the RTCPeerConnection so I could decrement `numGetters` when that happens, but couldn't find one.